prov/rxm/src/rxm_conn.c

```diff
@@ -1203,6 +1203,10 @@ rxm_conn_handle_event(struct rxm_ep *rxm_ep, struct rxm_msg_eq_entry *entry)
 		fi_freeinfo(entry->cm_entry.info);
 		break;
 	case FI_CONNECTED:
+		if (!entry->cm_entry.fid->context) {
+			FI_WARN(&rxm_prov, FI_LOG_FABRIC, "Null context in FI_CONNECTED event\n");
+			goto err;
+		}
```

Comment:
In manual progress mode, this would be in the critical path (called from rxm_ep_do_progress). Can we change this to an assert?

Reply:
That makes sense, I should have thought of that myself. Updated with assert instead of log.